src/www/ui_components/server_card/stories.ts

```diff
@@ -1,4 +1,20 @@
 /* tslint:disable */
+/*
+  Copyright 2022 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
 import {ServerConnectionState} from "./server_connection_viz";
```

Comment:
Use the definition from index.ts. The one from `server_connection_viz` should be treated as private and only accessed from index.ts.

Also the interface for the server-card is the one in index.ts, which just happens to be the same as the viz